.github/workflows/ci.yml

```diff
@@ -17,10 +17,13 @@ jobs:
       cancel-in-progress: true
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        with:
+          path: jco
 
       - name: Get Ballerina Version
         run: |
+          cd jco
```

Comment:
Shall we use working-dir config for an action

Reply:
Updated